src/SprykerEco/Zed/Unzer/Persistence/UnzerRepository.php

```diff
@@ -0,0 +1,306 @@
+    }
+
+    /**
+     * @module Store
+     *
+     * @param \Orm\Zed\Unzer\Persistence\SpyUnzerCredentialsQuery $unzerCredentialsQuery
+     * @param \Generated\Shared\Transfer\UnzerCredentialsConditionsTransfer $unzerCredentialsConditionsTransfer
+     *
+     * @return \Orm\Zed\Unzer\Persistence\SpyUnzerCredentialsQuery
+     */
+    protected function setUnzerConfigFilters(
+        SpyUnzerCredentialsQuery $unzerCredentialsQuery,
+        UnzerCredentialsConditionsTransfer $unzerCredentialsConditionsTransfer
+    ): SpyUnzerCredentialsQuery {
+        if ($unzerCredentialsConditionsTransfer->getKeypairIds()) {
+            $unzerCredentialsQuery->filterByKeypairId_In($unzerCredentialsConditionsTransfer->getKeypairIds());
+        }
+
+        if ($unzerCredentialsConditionsTransfer->getMerchantReferences()) {
+            $unzerCredentialsQuery->filterByMerchantReference_In($unzerCredentialsConditionsTransfer->getMerchantReferences());
+        }
+
+        if ($unzerCredentialsConditionsTransfer->getPublicKeys()) {
+            $unzerCredentialsQuery->filterByPublicKey_In($unzerCredentialsConditionsTransfer->getPublicKeys());
+        }
+
+        if ($unzerCredentialsConditionsTransfer->getTypes()) {
+            $unzerCredentialsQuery->filterByType_In($unzerCredentialsConditionsTransfer->getTypes());
+        }
+
+        if ($unzerCredentialsConditionsTransfer->getIds()) {
+            $unzerCredentialsQuery->filterByIdUnzerCredentials_In($unzerCredentialsConditionsTransfer->getIds());
+        }
+
+        if ($unzerCredentialsConditionsTransfer->getParentIds()) {
+            $unzerCredentialsQuery->filterByParentIdUnzerCredentials_In($unzerCredentialsConditionsTransfer->getParentIds());
+        }
+
+        if ($unzerCredentialsConditionsTransfer->getStoreNames()) {
+            $unzerCredentialsQuery
+                ->useUnzerCredentialsStoreQuery()
+                    ->joinWithStore()
+                    ->useStoreQuery()
+                        ->filterByName_In($unzerCredentialsConditionsTransfer->getStoreNames())
+                    ->endUse()
+                ->endUse();
+        }
+
+        if ($unzerCredentialsConditionsTransfer->getStoreIds()) {
+            $unzerCredentialsQuery
+                ->useUnzerCredentialsStoreQuery()
+                    ->joinWithStore()
+                    ->useStoreQuery()
+                        ->filterByIdStore_In($unzerCredentialsConditionsTransfer->getStoreIds())
+                ->endUse()
+                ->endUse();
+        }
+
+        return $unzerCredentialsQuery;
+    }
+
+    /**
+     * @param \Orm\Zed\Unzer\Persistence\SpyPaymentUnzerTransactionQuery $paymentUnzerTransactionQuery
+     * @param \Generated\Shared\Transfer\PaymentUnzerTransactionConditionsTransfer $paymentUnzerTransactionConditionsTransfer
+     *
+     * @return \Orm\Zed\Unzer\Persistence\SpyPaymentUnzerTransactionQuery
+     */
+    protected function applyPaymentUnzerTransactionConditions(
```

Comment:
please use consistent naming
method above has `setXXXFilters` naming
and here you have `applyXXXconditions`